op-e2e/actions/interop/interop_test.go

```diff
@@ -220,6 +224,129 @@ func TestFinality(gt *testing.T) {
 	})
 }
 
+func TestInteropLocalSafeInvalidation(gt *testing.T) {
```

Comment:
nit: I'd like if we started moving large, targeted tests into their own files so that `interop_test.go` isn't thousands of lines covering many topics.

Reply:
I'm going to leave it where it is for now, since proofs team is looking at this test, and other tests are already there. We can do a sweep of test reorganization once there's less in-flight